scripts/discovery.py

```diff
@@ -40,6 +40,7 @@
         help="1 for mainnet, 3 for testnet"
     )
     parser.add_argument('-l', type=str, help="Log level", default="info")
+    parser.add_argument('-enrdb', type=str, help="Path to ENR database", default="/tmp/enrs")
```

Comment:
```suggestion
    parser.add_argument('--enrdb', type=str, help="Path to ENR database", default="/tmp/enrs")
```

Even more nitpicky: Looking through the existing Trinity options, the naming convention seems to prefer dashes to separate words (so `--enr-db`).

Reply:
Well, this is just a helper script and all other arguments here use single dash, so I'd rather stick to that for now